howto/instance/access-instance.md

```diff
@@ -1,9 +1,66 @@
 (howto-access-instance)=
 # How to access an instance
 
-In some cases, it might be necessary to access an individual instance for debugging reasons.
+In some cases, it might be necessary to access an individual instance for debugging or development purposes.
```

Comment:
Can we add some specific use-cases?

"In some cases, it might be necessary to access an individual instance for debugging or developmental purposes. You might require to view the instance logs .... ..... etc."

Reply:
We discussed this, this will taken care with the separation of content related to Anbox instance vs Android instance.

Reply:
+1.